themes/docsy/layouts/partials/toc.html

```diff
@@ -2,7 +2,7 @@
 {{ if not .Params.notoc }}
 {{ with .TableOfContents }}
 {{ if ge (len .) 200 }}
-<h5 class="mb-4">On This Page</h5>
+<h5 class="mt-6 mb-4">On This Page</h5>
```

Comment:
Can it also be aligned with the stuff under it? It's further to the left. Thanks.

Reply:
Yeah I think the quote box comes with an indent so maybe that's what's showing up?